installer/bundle/bundle_skel_Linux.sh

```diff
@@ -749,19 +624,20 @@ case "$installMode" in
             check_if_pkg_is_installed omi
             if [ $? -eq 0 ]; then
                 shouldInstall_omi
-                pkg_upd $OMI_PKG omi $?
-                # It is acceptable that this fails due to the new omi being 
-                # the same version (or less) than the one currently installed.
-                OMI_EXIT_STATUS=0
+                pkg_upd_list $OMI_PKG omi $?
             else
-                pkg_add $OMI_PKG omi
-                OMI_EXIT_STATUS=$?  
+                pkg_add_list $OMI_PKG omi
             fi
         fi
 
         shouldInstall_scx
-        pkg_upd $OM_PKG scx $?
-        SCX_EXIT_STATUS=$?
+        pkg_upd_list $OM_PKG scx $?
+
+        pkg_add ${ADD_PKG_QUEUE}
+        OMI_EXIT_STATUS=$?
```

Comment:
Same messyness as above. These two lines should probably go right after 629.

Frankly, I'm not entirely sure you should do a separate test here (installing initially via upgrade isn't an error for RPM). But if I were you, I'd just want to get this wrapped up, so don't bother removing the 'if' test at 625 if you don't want to. I'll leave that up to you.